### DISCUSSION TITLE
Why isn't Elasticsearch displayed in the topology diagram

### DISCUSSION THREAD

**Discussion by 2han9b1n** [apache/skywalking#12856] [Q&A]
Why doesn't  Elasticsearch display an icon in the service topology diagram? Check if the interface type is empty

**Comment by wu-sheng** [comment]
I guess you are using Java agent. You need to check whether your elasticsearch access is traced.

**Reply by 2han9b1n** [reply]
Do I need to open this  【plugin.elasticsearch.trace_dsl】

**Reply by wu-sheng** [reply]
No, don't need to worry for that.

Do you check which library of elasticsearch client are you using? Whether it is listed here, https://skywalking.apache.org/docs/skywalking-java/latest/en/setup/service-agent/java-agent/supported-list/

**Reply by 2han9b1n** [reply]
Doesn't it support version 8.x?

**Reply by wu-sheng** [reply]
By looking at the docs, it seems so. Plugins are various, they are from global contributors.
